src/main.js

```diff
@@ -35,53 +30,77 @@ process.on('uncaughtException', (exception: Error) => {
  */
 const container = new Container();
 defaultBindings(container, settings);
-
 const deviceServer = container.constitute('DeviceServer');
-deviceServer.start();
-
 const app = createApp(container, settings);
 
-const onServerStartListen = (): void =>
-  console.log(`express server started on port ${NODE_PORT}`);
+const startServers = () => {
+  deviceServer.start();
```

Comment:
How this works now: we have master + x(forks_count) forks. The constitute code runs for `all` of those(forks+master), but servers *starts* only for forks. If I move constitute code inside startServers() function, it will start  server in  master process, too and I'll have to write additional shit in eventPublisher, to send events from master EventPubisher(basically, I'll need to duplicate what I have inside that `hub` code into `publish` function with `isMaster` check)